### PR TITLE
Add new guardian styles, upgrade dependencies.

### PR DESCRIPTION
**Pull request by philmcmahon**
This is what it looks like now - not perfect but at least a little more on brand.

screenshot from 2018-10-08 13-02-32

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
+        headline: content.fields.headline,
+        trailText: content.fields.trailText,
+        mainImage: mainImage
     }
 }
 
 function addRowOfNews(rowSubject, numberOfItems){
-    var contentApiQuery = encodeURIComponent(rowSubject);
-    return addContentList(rowSubject, "http://content.guardianapis.com/search?q="+contentApiQuery, numberOfItems);
+    const contentApiQuery = encodeURIComponent(rowSubject);
```
Comment (akash1810) [inline script.js]: `const` isn't supported by IE10-. Is it safe to assume the page will be viewed in a modern browser?

File: script.js [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+    return addContentList(rowSubject, `http://content.guardianapis.com/search?q=${contentApiQuery}`, numberOfItems);
```
Comment (akash1810) [inline script.js]: Template strings [aren't supported in IE](https://caniuse.com/#feat=template-literals). Is it safe to assume the page will be viewed in a modern browser?

**Review by akash1810 — APPROVED** [review]
A couple of comments around browser support. If we're happy to target modern Chrome/FF only, then 👍

File: script.js (hunk 1 above)
Comment (philmcmahon) [reply script.js]: Good point, might as well support IE 10 since the rest of the code seems ok

**Pull request merged**